rasa_nlu/config.py

```diff
@@ -111,6 +111,15 @@ def __init__(self, configuration_values=None):
             pipeline = registry.pipeline_template(template_name)
 
             if pipeline:
+                new_names = {"spacy_sklearn": "pretrained_embeddings_spacy",
+                             "tensorflow_embedding": "supervised_embeddings"}
+                if template_name in new_names:
```

Comment:
do we not also need to set the `template_name` from e.g. `"spacy_sklearn"` to `pretrained_embeddings_spacy`?

Reply:
yeah you're right, better to throw a warning and replace the name than to throw an error and quit.